recipe/meta.yaml

```diff
@@ -30,6 +30,8 @@ build:
     - '*/libX11.so*'              # [linux]
     - '*/libXext.so*'             # [linux]
     - '/usr/lib/libcups.2.dylib'  # [osx]
+  ignore_run_exports:
+    - pixman
 
 requirements:
   build:
```

Comment:
I think it's better to add pinnings for reliability, see https://gitlab.gnome.org/GNOME/gtk/-/blob/main/meson.build#L13-23

Reply:
Do we need `python >=3` in `req/build`? see https://gitlab.gnome.org/GNOME/gtk#building-and-installing